staging/src/k8s.io/code-generator/cmd/conversion-gen/args/args.go

```diff
@@ -57,13 +59,20 @@ type Args struct {
 	// GoHeaderFile is the path to a boilerplate header file for generated
 	// code.
 	GoHeaderFile string
+
+	// GeneratedBuildTag is the tag used to identify code generated by execution
```

Comment:
I don't like the "should be unique" language because we DO NOT do that in kubernetes.  Should we?  I don't know -- it's working pretty well as it is.

Reply:
Honestly, I copied/pasted this over from gengo args code. This is the language they use over there. It should be doesn't mean it has to be though so I think we are OK. Otherwise, we would have used the phrase "must be".